Installation/Action/waar.WaardepapierenAction.action.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "WaardepapierenAction",
+  "$id": "https://waardepapieren.commonground.nl/action/waar.WaardepapierenAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+  "version": "0.0.1",
+  "listens": [
+    "commongateway.object.created"
+  ],
+  "description": "This is a action to validate a certificate.",
+  "throws": [],
+  "conditions": {
+    "==": [{"var": "entity"}, "https://waardepapieren.commonground.nl/certificate.schema.json"]
+  },
+  "class": "CommonGateway\\WaardepapierenBundle\\ActionHandler\\WaardepapierenHandler",
+  "configuration": {},
+  "isLockable": false,
+  "isEnabled": true
```

Comment:
This is fine, but just so you know, isLockable is by default false and isEnabled is by default true